ui/src/app/views/homepage/component.html

```diff
@@ -23,43 +23,45 @@
       </div>
     </div>
   </div>
-  <div class="footer">
-    <div class="row" style="display: flex; flex-direction:column; margin-left:5%; margin-right: 5%;">
-      <div class="footerTitle">How to Use the All of Us Researcher Workbench</div>
-      <div class="footerRtLink">
-        <div class="tooltip">
-          <a href="#" class="footerLink">
-            See all documentation</a>
-          <span class="tooltiptext">Coming Soon</span>
+  <div #fullFooter>
```

Comment:
opt_nit: may not need this ID here since it doesn't seem to be getting used anywhere.